frontend/ui/widget/touchmenu.lua

```diff
@@ -465,14 +465,16 @@ function TouchMenu:init()
         -- Add some span to balance up_button image included padding
         HorizontalSpan:new{width = Size.span.horizontal_default},
     }
+    local footer_width = self.width - self.padding*2
     local up_button = IconButton:new{
         icon_file = "resources/icons/appbar.chevron.up.png",
         show_parent = self.show_parent,
+        padding_left = footer_width*0.33*0.1,
```

Comment:
Wouldn't it make more sense to base it on the IconButton itself or is that not known yet?

Reply:
Size of Icon Button doesn't know yet. Beside of that better to set up left and right padding to 3,3% of footer size.